### DISCUSSION TITLE
WSJ haven’t been updated since 21st March

### DISCUSSION THREAD

**Discussion by ajzlavie** [agorahub/_meta#17] [Q&A] [answered]
Hello Agorahub,
Many thanks for sharing this amazing news hub, and special thanks for this great contribution! I found WSJ haven’t been updated since 21st March, does anyone know what is going on, and any possible solutions? Thank you!
regards,

AJZ

**Comment by theagora** [comment, marked as answer]
Alternative sources: (in case the wsj source N.A. for >1wk)
- [ ] ~ftchinese.com, or~
- [x] cn.nytimes.com (2024.4.1~)

Proposal: (as wsj source [feedx/wsj](https://feedx.net/rss/wsj.xml) works again, 2024.4.6~)
* Updated 2024.11.4
- [x] ~`WSJ`, listed it in the navigation bar (2024.4.8-2024.8)~
- [x] ~`ABC`, unlisted, merged with `AJ` (2024.4.8-2024.8)~
- [x] `NYT`, added to replace `WSJ` (2024.4.8~)
- [x] `NHK`, added to replace `BBC` (2024.11.1~)

**Reply by ajzlavie** [reply]
Hi theagora,  

Thanks for the reply and thank you! I would say WSJ China is the best news website, they hire many dedicated local staffs in HK, Singapore to cover tech, energy, finance and other segments in APAC. The news they produced is immediate, concrete, and accurate. This is the major advantages for WSJ China. 

If you ask me on alternatives, ftchinese is worse compared with nytimes, but I will not recommended them both and PLEASE do not do any programming work since we already have 8 rich news websites on 零新聞, the resources is quite sufficient and thank you so much and sharing. 

But I would persist in reading WSJ and let wait and see what is going on next. 

Have a good day!
Regards,

AJZ

**Reply by theagora** [reply]
We can turn back to the wsj source once it works again for >1wk.
Let's watch, e.g., [feedx/wsj](https://feedx.net/rss/wsj.xml).

**Reply by ajzlavie** [reply]
Alright, and let see how to proceed next, thank you and have a nice day!

**Reply by ajzlavie** [reply]
Hello theagora, the WSJ still not updated on the defined frequency

**Reply by theagora** [reply]
The wsj source feed [feedx/wsj](https://feedx.net/rss/wsj.xml) works now. Let's see whether to turn it on again for the news0.

**Comment by pptanalysis** [comment]
Dear agora,

I rely on accessing "WSJ" via your website almost daily. Please ensure swift resolution. Thank you.
